app/src/main/java/xyz/whereuat/whereuat/Constants.java

```diff
@@ -24,4 +24,7 @@
     public static final String DEFAULT_PHONE_REGION = "US";
 
     public static final String CLIENT_OS = "ANDROID";
+
+    public static final String FROM_KEY = "from-#";
+    public static final String PLACE_KEY = "place";
```

Comment:
Maybe make it more explicit that these are used in GCM? Something like `GCM_FROM_KEY`, `GCM_PLACE_KEY`?